### PR TITLE
Funny bolter man

### PR DESCRIPTION
**Pull request by reklaw9**
this probably won't break anything

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/mattguns.dm [hunk 1]
```diff
@@ -725,8 +725,7 @@
 	fire_delay = 3.5
 	burst = 1
 	move_delay = 5
-	one_hand_penalty = 8
```
Comment (LeadOnTaste) [inline code/modules/projectiles/guns/projectile/mattguns.dm:728]: Don't touch that var ever

File: code/modules/projectiles/guns/projectile/mattguns.dm [hunk 2]
```diff
@@ -792,18 +790,17 @@
...
-	one_hand_penalty = 9
+	one_hand_penalty = 7
 	fire_sound = 'sound/weapons/gunshot/harbinger.ogg'
 	unload_sound 	= 'sound/weapons/guns/interact/ltrifle_magout.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/ltrifle_magin.ogg'
 	cock_sound 		= 'sound/weapons/guns/interact/ltrifle_cock.ogg'
 	loaded_icon = "sallybolter-30"
 	unloaded_icon = "sallybolter-e"
-	fire_delay = 3.5
+	fire_delay = 2 //salamanders love to fix their weapons and their guns normally are master crafted
```
Comment (LeadOnTaste) [inline code/modules/projectiles/guns/projectile/mattguns.dm:800]: It does not adds RPM, they just craft shit. Put it back.

File: code/modules/projectiles/guns/projectile/mattguns.dm [hunk 3]
```diff
@@ -792,18 +790,17 @@
...
 	burst = 1
 	move_delay = 5
-	one_hand_penalty = 4
```
Comment (LeadOnTaste) [inline code/modules/projectiles/guns/projectile/mattguns.dm:805]: I said: DO NOT TOUCH THAT FUCKING VAR

**Review by LeadOnTaste — CHANGES_REQUESTED** [review]
No, no, no, I approve Locke ST requirement only.

**Comment by reklaw9** [comment]
alright one moment

**Pull request closed without merging**